.pre-commit-config.yaml

```diff
@@ -213,7 +213,9 @@ repos:
       - id: pyupgrade
         name: Upgrade Python code automatically
         args: ["--py37-plus"]
-        exclude: ^airflow/_vendor/
+        # We need to exclude gcs hook from pyupgrade because it has public "list" command which clashes
+        # with `list` that is used as type
```

Comment:
Is this a bug in pyupgrade?

Reply:
Not really. This is a problem with naming our function "list". Simply Python thinks that "list" used as return type is our method.

Reply:
Similarly as in the other case, I just let pyupgrade do the job and reverted the list change

Reply:
Similarly as in the other case, I just let pyupgrade do the job and reverted the list change